.gitlab-ci.yml

```diff
@@ -62,14 +64,47 @@ azure:
     - |
       sudo podman run \
       -a stdout -a stderr \
-      -e ARM_CLIENT_ID="${AZURE_CLIENT_ID}" \
-      -e ARM_CLIENT_SECRET="${AZURE_CLIENT_SECRET}" \
+      -e ARM_CLIENT_ID="${V2_AZURE_CLIENT_ID}" \
+      -e ARM_CLIENT_SECRET="${V2_AZURE_CLIENT_SECRET}" \
       -e ARM_SUBSCRIPTION_ID="${AZURE_SUBSCRIPTION_ID}" \
       -e ARM_TENANT_ID="${AZURE_TENANT_ID}" \
       -v ./:/tmp:Z \
       "${QUAY_IO_CONTAINER_URL}":"${CI_COMMIT_REF_SLUG}" \
       python cloud-image-val.py -r cloud/sample/resources_azure_marketplace.json -d -p -o /tmp/report.xml
 
+.rhel_runners: &rhel_runners
+    RUNNER:
+      - aws/centos-stream-8-x86_64
+      - aws/rhel-8.4-ga-x86_64
+      - aws/rhel-8.6-ga-x86_64
+      - aws/rhel-9.0-ga-x86_64
+      - aws/rhel-8.7-nightly-x86_64
+      - aws/rhel-9.1-nightly-x86_64
+      - aws/centos-stream-9-x86_64
```

Comment:
I thick centos should be in .centos_runners

Reply:
centos is included in this category as it is considered a rhel like system, unlike fedora. I think the only reason this rhel-fedora separation is made in the yaml is to give acces to rhel like systems to the internal network. But we can separate centos